go/chat/context.go

```diff
@@ -105,7 +110,10 @@ func Context(ctx context.Context, g *globals.Context, mode keybase1.TLFIdentifyB
 	if _, ok := val.(KeyFinder); !ok {
 		res = context.WithValue(res, kfKey, NewKeyFinder(g))
 	}
-	res = context.WithValue(res, inKey, notifier)
+	val = res.Value(inKey)
```

Comment:
This was useful for testing, just keep the `IdentifyNotifier` if it is already on the context.